pyproject.toml

```diff
@@ -74,6 +74,8 @@ Werkzeug = "<2.0.0"
 click = "<8.0.0"
 sentry-sdk = "<1.6.1"
 pytest-invenio = ">=1.4.0,<1.4.12"
+jsonref = "<1.0.0"
```

Comment:
Did you use `poetry add` to avoid all update dependencies?

Reply:
No I wanted to update the other packages too.